.github/workflows/dev_deploy.yml

```
@@ -0,0 +1,105 @@
+  pull_request:
+    types: [closed]
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    # if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)